apps/ima/translations/src/en/pages.json

```diff
@@ -63,29 +60,31 @@
     "header": "Declaration"
   },
   "immigration-adviser-declaration": {
+    "adviser": "Immigration adviser declaration",
     "header": "Declaration",
     "paragraph1": "What language was used to read the completed form to {{values.name}}?",
-    "paragraph2": "Include any dialects",
-    "header1": "Did you use an interpreter?"
+    "paragraph2": "Include any dialects"
   },
   "helper-declaration": {
+    "helper": "Helper declaration",
     "header": "Declaration",
     "paragraph1": "What language was used to read the completed form to {{values.name}}?",
-    "paragraph2": "Include any dialects",
-    "header1": "Did you use an interpreter?"
+    "paragraph2": "Include any dialects"
+  },
+  "summary": {
+    "summary": "Summary"
   },
   "form-submitted": {
-    "confirmed": "Form-Submitted",
+    "confirmed": "Form Submitted",
     "paragraph1": "We have sent a copy of completed form to:",
     "header2": "What happens next",
     "paragraph2": "We will make decision about your claim, then contact you to tell you what you need to do next.",
     "header3": "If you need to send more evidence",
-    "paragraph3": "You can email evidence to [EMAIL ADDRESS]",
-    "paragraph4": "Your email must include the Unique application number [UAN] and date of birth you used to sign in to this form.",
-    "paragraph5": "Information about how to send more evidence is also on the email we send you when you submitted this form.",
+    "paragraph3": "If you have given a claim period and need to send evidence after the end of the period, you need to:",
+    "paragraph4": "1. apply for an extension at the end of your claim period",
+    "paragraph5": "2. send your evidence to [EMAIL ADDRESS] as soon as possible",
     "paragraph6": "Do not wait for confirmation that you have been granted an extension to send your evidence.",
     "paragraph7": "Your email subject line must contain the Unique application number {{values.uan}} and date of birth you used to sign in to this form.",
     "paragraph8": "Information about how to send more evidence is also on the email we send you when you submitted this form."
-
```

Comment:
I'd put this in a markdown

Reply:
New markdown created and applied css style.

Reply:
Once you have put this in a markdown it should be removed from the pages.json

Reply:
header3 added to markdown  with ### and removed from html.